client/state/invites/selectors.js

```diff
@@ -100,7 +101,7 @@ export const getInviteForSite = treeSelect(
  * @returns {boolean}          Whether invites resend is being requested
  */
 export function isRequestingInviteResend( state, siteId, inviteId ) {
-	return 'requesting' === get( state, [ 'invites', 'requestingResend', siteId, inviteId ], false );
+	return 'requesting' === state.invites.requestingResend[ siteId ]?.[ inviteId ];
```

Comment:
Should we continue falling back to `false`?

```suggestion
	return 'requesting' === state.invites.requestingResend[ siteId ]?.[ invited ] ?? false;
```

Reply:
I'm open to changing it but I don't think it makes a difference. Ultimately we're comparing `'requesting' === undefined` vs. `'requesting' === false` in this case and both will evaluate to `false`.

Reply:
That's fair, let's leave it as is 👍